Samples/Windowing/cs-wpf/wpf_packaged_app/wpf_packaged_app.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
+    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
+    <Platforms>x86;x64</Platforms>
+    <RuntimeIdentifiers>win-x86;win-x64</RuntimeIdentifiers>
+    <UseWPF>true</UseWPF>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.0.0-preview1" />
```

Comment:
Might need <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.20348.19" />
@rohanp-msft to confirm

Reply:
We do want Microsoft.Windows.SDK.BuildTools to be used. I confirmed. (When the package is installed in the app, tools like makepri.exe and makeappx.exe are used from the package rather than the Windows SDK installed on the machine.)

The thing is that the package may already be used here. It should be brought in as a dependency of the WAS package, but @erik0686 found in https://github.com/microsoft/WindowsAppSDK-Samples/pull/81 that it doesn't get added explicitly sometimes... He manually added the package where it wasn't present explicitly, which now I think perhaps we shouldn't have done.

Let me download this change and test to find out if the Microsoft.Windows.SDK.BuildTools package is being used already without an explicit reference.

Either the package reference should be added explicitly always or implicitly always. If it's a mix, that's a bug and should be reported. That's separate from this change though, of course.

Reply:
@samanthamsong 
I found that Microsoft.Windows.SDK.BuildTools _is_ already being used in both the CSPROJ and the WAPPROJ projects.
I say don't add an explicit `PackageReference`.
I'll see if I can repro an explicit reference being added in other cases. If I can, I'll file a bug.